### PR TITLE
281 add access link

### PR DESCRIPTION
**Pull request by abrahimi93**
Now, users will be able to have access code saved via a link. 

fixes https://github.com/hobbyfarm/hobbyfarm/issues/281

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -119,11 +124,21 @@ export class AppComponent implements OnInit {
...
     const timeout = tok.exp * 1000 - Date.now();
     setTimeout(() => this.doLogout(), timeout);
 
+    this.accesscode = this.route.snapshot.params['accesscode'];
+    if( this.accesscode ){
+      this.newAccessCodeForm.setValue({access_code: this.accesscode});
+      this.setAccessCode(this.accesscode);
+      this.saveAccessCode();
+      this.doHome();
+      this.accessCodeLinkSuccessAlert = `${this.accesscode} added`;
```
Comment (jggoebel) [inline src/app/app.component.ts:139]: This is not executed anymore after you call doHome, or is it?

File: src/app/home.component.ts [hunk 2]
```diff
@@ -83,6 +90,16 @@ export class HomeComponent implements OnInit, OnDestroy {
       // thus, refresh the scenario list
       this.contextService.refresh();
     });
+
+    this.accesscode = this.route.snapshot.queryParams['ac'];
+    if( this.accesscode ){
+      this.accessCodeLinkSuccessAlert = `${this.accesscode} added`;
+      this.accessCodeLinkSuccessClosed = false;
+      this.location.go("/app/home");
+      setTimeout(() => {
```
Comment (jggoebel) [inline src/app/home.component.ts:99]: Why are you reloading the page?
Is it not possible to just remove the accessCodeLinkSuccessAlert again or set it to closed?

File: src/app/app.component.ts (hunk 1 above)
Comment (abrahimi93) [reply src/app/app.component.ts:139]: Oh, I didn't pay attention to this dead code. I will take care of it.

File: src/app/home.component.ts (hunk 2 above)
Comment (abrahimi93) [reply src/app/home.component.ts]: It's done. I set it to closed, but I nested inside setTimeout 5 second delay

**Pull request merged**